driver/config/.schema/config.schema.json

```
@@ -149,6 +149,11 @@
               "type": "string",
               "description": "Path to the jsonnet template used for payload generation. Only used for those HTTP methods, which support HTTP body payloads"
             },
+            "interrupt": {
```

Let's make this explicit:

```suggestion
            "can_interrupt": {
```